srcpkgs/linux5.15/patches/fix-lenovo7i-sound.patch

```diff
@@ -0,0 +1,12 @@
+diff -Naur sound/pci/hda/patch_realtek.c sound/pci/hda/patch_realtek-new.c
```

Comment:
The patch should mention its upstream status.

Reply:
Updated.